scripts/python/services/GithubService.py

```diff
@@ -25,17 +26,19 @@ def decorator(*args, **kwargs):
             - Deleting data
             - Updating data
         """
-        github_client_core_api = args[0].github_client_core_api
         try:
             return func(*args, **kwargs)
-        except RateLimitExceededException:
-            logging.warning(
-                "Caught RateLimitExceededException, retrying calls when rate limit resets.")
-            core_api_reset_timestamp = timegm(
-                github_client_core_api.get_rate_limit().core.reset.timetuple())
+        except (RateLimitExceededException, TransportQueryError) as exception:
```

Comment:
`TransportQueryError` was taken from the [output on this job](https://github.com/moj-analytical-services/operations-engineering/actions/runs/4300558586/jobs/7496858145#step:5:71)